### ISSUE TITLE
remix-graphql ships `fs` to the browser

### ISSUE THREAD

**Issue by alexluong** [thomasheyenbrock/remix-graphql#1]
Hi, I'm trying to use `remix-graphql` but it explodes the bundle size (from 150kB to 650kB).

To reproduce:

1. use `create-remix` to generate a new Remix app
2. use `sendGraphQLRequest` from `remix-graphql` in the loader

---

As a workaround, I re-export everything from `remix-graphql` in a `remix-graphql.server.ts` file so it will be excluded from the client bundle.

**Comment by thomasheyenbrock** [comment]
Hello @alexluong and thanks for opening this issue! I also stumbled over this while building `remix-graphql`, and I baked in the same "workaround" that you did into the library 😄 You can change your imports like this to tell remix to not include anything from `remix-graphql` in the browser bundle:

```diff
- import { sendGraphQLRequest } from "remix-graphql"; 
+ import { sendGraphQLRequest } from "remix-graphql/index.server"; 
```

The examples in the README use this import path, but it's never called out explicitly (which we should). It's also easy to forget when using auto-imports in the IDE, don't know if there's any way to mitigate that though.

If you have ideas, contributions are very welcome! ❤️ (I didn't set up anything like a contributing guide, CI, tests, PR templates or stuff yet, but that'll come if the library gains more traction.)

**Comment by thomasheyenbrock** [comment]
I created the PR linked above to help with this ☝️

**Comment by thomasheyenbrock** [comment]
Published in 0.2.1
